packages/app/src/app.ts

```diff
@@ -2,149 +2,149 @@ import createClient from "openapi-fetch";
 
 import { paths, components } from "./schema";
 import {
-    Notice,
-    Report,
-    AdvanceRequestHandler,
-    RequestHandlerResult,
-    Voucher,
-    AdvanceRequestData,
-    InspectRequestHandler,
-    InspectRequestData,
+  Notice,
```

Comment:
Why are these formatting differences here?
Prettier configuration should be 4 spaces.

Reply:
@SandHilt neovim?

Reply:
linter, prettier

Reply:
lgtm!